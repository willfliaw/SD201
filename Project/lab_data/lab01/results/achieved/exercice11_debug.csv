0.829268
0.984848
