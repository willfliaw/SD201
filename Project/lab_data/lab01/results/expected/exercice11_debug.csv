0.742857
0.984925
